src/SprykerEco/Zed/Adyen/AdyenConfig.php

```diff
@@ -304,4 +324,23 @@ public function getMapperPaymentMethods(): array
             static::PAYMENT_METHOD_TYPE_WE_CHAT_PAY => SharedAdyenConfig::ADYEN_WE_CHAT_PAY,
         ];
     }
+
+    /**
+     * @return string[]
+     */
+    public function getBankTransferPaymentMethods(): array
```

Comment:
Doesn't look like payment methods as for me :)

Reply:
What does it actually mean `banktransfer_DE` and `banktransfer_IBAN`? IBAN is international bank account number, DE is country code, how these different things can be in the same line?

Reply:
According to documentation and the payment methods settings in the Adyen's Spryker account, there are different `Bank Transfer` methods.
So, `IBAN` as you've mentioned (international BAN) is used for some list of countries (including `DE`). Also, there is an additional payment method is set for `DE` country is `banktransfer_DE`. And payments for `DE` store do not work using the `banktransfer_IBAN`  payment method. But for `banktransfer_DE` they work.